### PR TITLE
EclProblem: put properties in separate header

### PR DESCRIPTION
**Pull request by akva2**
No functional change but in my opinion cleaner to have these in a separate header.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

File: ebos/eclproblem_properties.hh [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+
+#include <opm/input/eclipse/Parser/ParserKeywords/E.hpp>
+
+#include <opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp>
+#include <opm/material/thermal/EclThermalLawManager.hpp>
+
+#include <opm/models/discretization/ecfv/ecfvdiscretization.hh>
+#include <opm/models/utils/propertysystem.hh>
+
+#if USE_ALUGRID
```
Comment (atgeirr) [inline ebos/eclproblem_properties.hh:48]: This part should be gone now.

**Review by atgeirr — COMMENTED** [review]
Ok to merge after removing the USE_ALUGRID things.

File: ebos/eclproblem_properties.hh (hunk 1 above)
Comment (akva2) [reply ebos/eclproblem_properties.hh]: rebase meh, fixed.

**Pull request merged**